osu.Game/Screens/OnlinePlay/Multiplayer/Match/MatchStartControl.cs

```diff
@@ -198,6 +219,8 @@ private void updateState()
             if (localUser?.State == MultiplayerUserState.Spectating)
                 readyButton.Enabled.Value &= Client.IsHost && newCountReady > 0 && !Room.ActiveCountdowns.Any(c => c is MatchStartCountdown);
 
+            readyButton.Enabled.Value = true;
```

Comment:
What's this? Given the pains that the code block above goes through to set the value of `readyButton.Enabled.Value`, this looks very much like a leftover debug hard-enable to me...